app/providers.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+import { SessionProvider as NextAuthSessionProvider } from "next-auth/react"
```

Comment:
Oh interesting. You're looking at next-auth. I punted on trying to use next-auth for anything saml and went with iron-session.

Also, TIL, next-auth became https://authjs.dev/ and is now https://www.better-auth.com/